src/Components/Patient/PatientRegister.tsx

```diff
@@ -147,6 +147,8 @@ const initForm: any = {
   cluster_name: "",
   covin_id: "",
   is_vaccinated: "false",
+  number_of_doses: 0,
+  vaccine_name: "CoviShield",
```

Comment:
are we considering only covishield ?

Reply:
@divyagar

Reply:
@tomahawk-pilot  No this is the default value.